src/tigerbeetle.zig

```diff
@@ -385,6 +385,20 @@ pub const QueryFilterFlags = packed struct(u32) {
 pub const AccountFilter = extern struct {
     /// The account id.
     account_id: u128,
+    /// Filter by the `user_data_128` index.
+    /// Use zero for no filter.
+    user_data_128: u128,
+    /// Filter by the `user_data_64` index.
+    /// Use zero for no filter.
+    user_data_64: u64,
+    /// Filter by the `user_data_32` index.
+    /// Use zero for no filter.
+    user_data_32: u32,
```

Comment:
Why not add `ledger` too?

Reply:
Because it’s already filtered by the account_id.

Reply:
Ah right, thanks!